spec/models/miq_product_feature_spec.rb

```diff
@@ -4,7 +4,7 @@
 
 describe MiqProductFeature do
   before do
-    @expected_feature_count = 872
+    @expected_feature_count = 878
```

Comment:
I agree, it's fine to do this one more time... but yeah, if it's not providing value to you or anyone else who's adding product features, in a followup PR let's remove the `@expected_feature_count` variable and assertion... The only use case I can imagine for this check was if you used a duplicate identifier, and that is tested better in one of the other examples.

How do you test the product feature is in the database for use by the UI without manually firing up the UI and clicking around?  I guess that's the real way to test this.